go.mod

```diff
@@ -32,13 +32,13 @@ require (
 	k8s.io/api v0.20.2
 	k8s.io/apimachinery v0.20.2
 	k8s.io/client-go v12.0.0+incompatible
-	knative.dev/eventing v0.20.1
+	knative.dev/eventing v0.21.3
```

Comment:
Please update the KNATIVE_VERSION in `./hack/update-deps.sh`. I'd think we should get https://github.com/openshift-knative/serverless-operator/pull/882 in first, then bump the version and run `./hack/update-deps.sh --upgrade` again for full consistency.

Reply:
Okay